packages/shylu/shylu_dd/frosch/src/InterfaceSets/FROSch_DDInterface_def.hpp

```diff
@@ -687,12 +687,12 @@ namespace FROSch {
                     for (int i=0; i<NumMyNodes_; i++) {
                         commMat->insertGlobalValues(NodesMap_->getGlobalElement(i),myPID(),one());
                     }
-                    XMapPtr rangeMap = MapFactory<LO,GO,NO>::Build(NodesMap_->lib(),-1,myPID(),0,NodesMap_->getComm());
+                    XMapPtr domainMap = MapFactory<LO,GO,NO>::Build(NodesMap_->lib(),-1,myPID(),0,NodesMap_->getComm());
 
-                    commMat->fillComplete(NodesMap_,rangeMap);
+                    commMat->fillComplete(domainMap,NodesMap_);
                     commMatTmp->doExport(*commMat,*commExporter,INSERT);
-                    commMatTmp->fillComplete(UniqueNodesMap_,rangeMap);
-                    commMat = MatrixFactory<SC,LO,GO,NO>::Build(NodesMap_,10);
+                    commMatTmp->fillComplete(domainMap,UniqueNodesMap_);
+                    commMat = MatrixFactory<SC,LO,GO,NO>::Build(NodesMap_,as<LO>(0));
```

Comment:
You don't need `as` here.  Just do this: `LO(0)`.

Reply:
See above.